src/templates/directoryListing.hbs

```diff
@@ -1,31 +1,11 @@
 <html>
-  <head>
-    <title>Index of {{pathname}}</title>
-    <meta name='viewport' content='width=device-width, initial-scale=1.0' />
-    <meta charset='utf-8' />
-    <style type='text/css'>
-      td { padding-right: 16px; text-align: right; font-family: monospace }
-      td:nth-of-type(1) { text-align: left; overflow-wrap: anywhere }
-      td:nth-of-type(3) { white-space: nowrap } th { text-align: left; } @media
-      (prefers-color-scheme: dark) { body { color: white; background-color:
-      #1c1b22; } a { color: #3391ff; } a:visited { color: #C63B65; } }
-    </style>
-  </head>
+  <head><title>Index of {{pathname}}</title></head>
   <body>
-    <h1>Index of {{pathname}}</h1>
-    <table>
-      <tr>
-        <th>Filename</th>
-        <th>Modified</th>
-        <th>Size</th>
-      </tr>
+    <h1>Index of {{pathname}}</h1><hr /><pre><a href='../'>../</a><br />
       {{#each entries}}
-        <tr>
-          <td><a href='{{href}}'>{{name}}</a></td>
-          <td>{{lastModified}}</td>
-          <td>{{size}}</td>
-        </tr>
+        <a href='{{href}}'>{{name}}</a>
+        {{lastModified}}
+        {{size}}<br />
       {{/each}}
-    </table>
-  </body>
+    </pre><hr /></body>
```

Comment:
```suggestion
    </pre><hr></body>
```

Reply:
Afaik per HTML spec, `<hr>` should be a self-closing tag `<hr />`

Reply:
Self closing tags don't exist in html. The ending slash is ignored by parsers

Reply:
I don't think you're right. Self-closing tags are valid on "void elements" (https://html.spec.whatwg.org/multipage/syntax.html#void-elements)

Reply:
> On void elements, it does not mark the start tag as self-closing but instead is unnecessary and has no effect of any kind. For such void elements, it should be used only with caution — especially since, if directly preceded by an unquoted attribute value, it becomes part of the attribute value rather than being discarded by the parser.

Reply:
Right, but it's still valid HTML. Not arguing about being recommended or not, just your statement of being invalid HTML.

Reply:
I haven't said it's invalid.

Reply:
> Self closing tags don't exist in html.

That's what I understood as invalid 😅